tests/utils.py

```diff
@@ -67,6 +73,79 @@ def chdir(path):
     os.chdir(old)
 
 
+def add_dummy_language_plugin():
+    distribution = pkg_resources.Distribution(__file__)
+    entry_point = pkg_resources.EntryPoint.parse(
+        "dummy = rpdk.dummy:DummyLanguagePlugin", dist=distribution
+    )
+    distribution._ep_map = {  # pylint: disable=protected-access
+        "rpdk.v1.languages": {"dummy": entry_point}
+    }
+    pkg_resources.working_set.add(distribution)
+
+
+def get_mock_project():
+    mock_project = Mock(spec=Project)
+    mock_project.load_settings.side_effect = FileNotFoundError
+    mock_project.settings_path = ""
+    mock_project.root = Path(".")
+
+    patch_project = patch("rpdk.core.init.Project", return_value=mock_project)
+
+    return (mock_project, patch_project)
+
+
+def get_args(interactive=False, language=True, type_name=True):
+    args = Mock(
+        spec_set=[
+            "language",
+            "type_name",
+            "settings",
+        ]
+    )
+
+    args.language = (
+        None if interactive else ("dummy" if language else "invalid_language")
+    )
+    args.type_name = (
+        None if interactive else ("Test::Test::Test" if type_name else "Test")
```

Comment:
let's just pass in a type name into this get_args util instead of passing in boolean that sets between two values. makes the function more straightforward